components/automate-chef-io/content/docs/saml.md

```diff
@@ -42,12 +42,44 @@ Users who sign in via SAML will have a session time of 24 hours before needing t
 
 ## Supported Identity Management Systems
 
+- [Azure AD]({{< relref "#azure-ad" >}})
 - Office365
 - OKTA
 - OneLogin
 - Ping
 - Tivoli Federated Identity Manager
 
+In the following section, you can find further details about specific supported identity management systems.
+
+### Azure AD
+
+For using Chef Automate's SAML integration with Azure AD, please note that **signing key rotation** is
+not supported to be done automatically. The signing certificate used with Azure for SAML sign-ins
+has to be **managed manually**.
+
+After setting up your application as a _"non-gallery application"_, configure its SAML sign-on method.
```

Comment:
Are the fields in _italics_ supposed to be the relevant fields for the Azure AD set-up (and not directly for the Chef Automate interface)?

Reply:
Yeah! Is there another markup you'd like to have instead?

Reply:
Italics are fine. I just wanted to make sure that I was interpreting that those italics were for fields/labels since I can't reference them for myself. Thanks!